epictrack-web/src/components/myWorkplans/Card/CardBody.tsx

```diff
@@ -0,0 +1,149 @@
+import { Box, Grid, SxProps } from "@mui/material";
+import { Palette } from "../../../styles/theme";
+import { ETCaption1, ETCaption2, ETHeading4, ETParagraph } from "../../shared";
+import Icons from "../../icons";
+import { IconProps } from "../../icons/type";
+
+const IndicatorSmallIcon: React.FC<IconProps> = Icons["IndicatorSmallIcon"];
+const DotIcon: React.FC<IconProps> = Icons["DotIcon"];
+const ClockIcon: React.FC<IconProps> = Icons["ClockIcon"];
+
+const CardBody = () => {
+  return (
+    <Grid
+      container
+      direction="column"
+      justifyContent="flex-start"
+      sx={{
+        backgroundColor: Palette.white,
+        padding: "16px 24px",
+      }}
+      gap={1}
```

Comment:
one more ;p spacing={1}

Reply:
i had to leave this as gap, spacing breaks it